### PR TITLE
new translator ubtue_Asbury Journal & fixes issues 747

### PR DESCRIPTION
**Pull request by socheres**
https://github.com/ubtue/DatenProbleme/issues/742
https://github.com/ubtue/DatenProbleme/issues/747

### CODE REVIEW

File: ubtue_Tidsskrift.js [hunk 1]
```diff
@@ -35,46 +35,55 @@
...
 
-    let sidebarVals = ZU.xpath(doc, '//div[@class="value"]');
+	if (item.abstractNote) {
+		if (matchFirstAbstract) 
+			item.abstractNote = matchFirstAbstract[2].trim();
+		}
+	}
+	
+	var matchFirstAbstract = item.abstractNote.match(/\s.*/);
+	var matchSecondAbstract = item.abstractNote.match(/(DANSK RESUMÉ|DANSK RESUME).*/);
```
[the remark this answers is not in the document]
Comment (socheres) [reply ubtue_Tidsskrift.js]: Z.debug(item.abstractNote). It matches all strings after "DANSK RESUMÉ|DANSK RESUME"
`10:35:21 Resumé
         					ENGLISH ABSTRACT: This article discusses the nexus of religion and media on the basis of an analysis of one example of religion in popular culture: the expansion of J.K. Rowling’s Harry Potter series in the book Fantastic Beasts and Where to Find Them (2001) and the subsequent film by the same title connected to the book (directed by David Yates, screenplay by J.K. Rowling 2016). I present a study of religion-based, media-focused approach to film analysis that distinguishes between verbal and non-verbal aspects of mediation. The analysis treats the mediation of religion – traditional religion, magic and monstrous beings – in the film, as well as the mediality of the film. The analysis shows that the film forms part of a broader trend that portrays traditional religion as ossified and authoritative, while magic, monsters and green religion are represented as fascinating and attractive. Moreover, I argue that the mediality of the film sustains a blurring of boundaries between worlds in terms of the film-internal world structure, in terms of the diegetic vs. the afilmic world, and in terms of the hu-man vs. the nonhuman world. The key argument of the article is that pop-cultural me-dia constitute an important arena for religion, as media such as fantasy films both re-flect and form religious transformations today. This arena needs more attention in the study of religion.
         DANSK RESUME: Denne artikel diskuterer religion og medier ud fra en analyse af et eksempel på religion i populærkultur, nemlig en videreudvikling relateret til J.K. Row-lings Harry Potter univers: udgivelsen af bogen Fantastic Beasts and Where to Find Them (2001), en bog der er fiktivt indlejret som undervisningsbog på Hogwarts School of Wizardry and Witchcraft, og den senere filmatisering der knytter sig til bo-gen, Fantastic Beasts and Where to Find Them (instrueret af David Yates, screenplay af J.K. Rowling 2016). Artiklen præsenterer en religionsvidenskabelig, mediefokuseret tilgang til filmanalyse, der inkluderer både verbale og non-verbale medierings-aspekter. Analysen behandler fremstillingen af religion – hhv. traditionel religion, magi og det monstrøse – i det filmiske univers, samt filmens medialitet. Analysen viser, at filmen indlejrer sig i en bredere trend, hvor traditionel religion fremstilles som forstenet og autoritær, mens magi, monstre og grøn religion fremstilles positivt og tiltrækkende, samt endvidere at filmens medialitet understøtter en udviskning af grænserne mellem verdener både i den tekst-interne verdensstruktur, mellem den diegetiske og den afilmi-ske verden, samt mellem den humane og den non-humane verden. Artiklen argumente-rer grundlæggende for at populærkulturelle medier såsom fantasyfilm udgør en vigtig arena for religion, og at denne arena bør behandles religionsvidenskabeligt, da den både afspejler og former religiøse forandringer i samtiden.
10:35:21 Translate: WARNING: typeof abstractNote is object; converting to string`

File: Quaderni di storia religiosa medievale.js [hunk 2]
```diff
@@ -0,0 +1,274 @@
+{
+	"translatorID": "5db07685-2d24-4b0a-9659-2e6af73e58ba",
+	"label": "Quaderni di storia religiosa medievale",
+	"creator": "Timotheus Kim",
+	"target": "^https:?//www\\.rivisteweb\\.it/(doi|issn)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
```
Comment (mtrojan-ub) [inline Quaderni di storia religiosa medievale.js:9]: New translator => inRepository should be false, don't forget file name prefix "ubtue_"

File: ubtue_Asbury Journal [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+function detectWeb(doc, url) {
+    return "journalArticle";
+}
+
+function scrape(doc, item) {
+    item.DOI = text(doc, '#identifier p').replace('DOI:', '');
+    item.pages = text(doc, '.citation').match(/\d+-\d+/);
+    item.abstractNote = text(doc, '#abstract p');
+    let TagBookReview = text(doc, '#title a')
+	if (TagBookReview.match(/Book Reviews/)) {
```
Comment (mtrojan-ub) [inline ubtue_Asbury Journal:48]: strange indentation on lines 48-50

File: ubtue_Davarlogos.js [hunk 4]
```diff
@@ -70,33 +71,23 @@ function postProcess(doc, item) {
...
+	item.title = item.title.replace(' | DavarLogos', '');
 	item.volume = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]').trim().substr(4, 7).replace('Núm', '');
 	item.issue = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]').trim().substr(-20, 1);
 	item.date = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]').trim().substr(-17, 4);
 	item.url = attr(doc, '.pdf', 'href');
 	item.ISSN = '1666-7832'
-	item.itemType = 'journalArticle'
-	
-	item.complete();
+	//ToDO scrape item.pages before getSearchResults see e.g. fzpth
```
Comment (mtrojan-ub) [inline ubtue_Davarlogos.js:80]: Open TODO

**Comment by mtrojan-ub** [comment]
PS: Maybe you should set your editor to use 4 spaces instead of tabs?

File: Quaderni di storia religiosa medievale.js (hunk 2 above)
Comment (socheres) [reply Quaderni di storia religiosa medievale.js]: file name changed

File: ubtue_Davarlogos.js (hunk 4 above)
Comment (socheres) [reply ubtue_Davarlogos.js:80]: so solution found. you wanted to look for a workaround.

File: ubtue_Quaderni di storia religiosa medievale.js [hunk 5]
```diff
@@ -0,0 +1,274 @@
+{
+	"translatorID": "5db07685-2d24-4b0a-9659-2e6af73e58ba",
+	"label": "Quaderni di storia religiosa medievale",
+	"creator": "Timotheus Kim",
+	"target": "^https:?//www\\.rivisteweb\\.it/(doi|issn)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
```
Comment (mtrojan-ub) [inline ubtue_Quaderni di storia religiosa medievale.js:9]: inRepository should be false

**Comment by socheres** [comment]
@mtrojan-ub would you take a closer look at the errors. I fixed the errors as far as possible.

**Comment by mtrojan-ub** [comment]
@socheres this looks a lot better now!

It's uncritical to me if you do not fix all errors like "expected linebreak" and so on, but there are still 2 errors which look critical to me because these are real syntax errors and will prevent the whole translator from being executed, please have a look ("Parsing error"):
- ubtue_Journal of European Baptist Studies.js (line 38): Looks like a missing closing parenthesis ")" at the end of line 37
- ubtue_Tidsskrift.js (line 336): Now that's a harder one, i think the error is in fact that the doWeb function has no ending, so there is a missing parenthesis "}" around line 105 (before the test cases start).

**Comment by socheres** [comment]
> @socheres this looks a lot better now!
> 
> It's uncritical to me if you do not fix all errors like "expected linebreak" and so on, but there are still 2 errors which look critical to me because these are real syntax errors and will prevent the whole translator from being executed, please have a look ("Parsing error"):
> 
> * ubtue_Journal of European Baptist Studies.js (line 38): Looks like a missing closing parenthesis ")" at the end of line 37
> * ubtue_Tidsskrift.js (line 336): Now that's a harder one, i think the error is in fact that the doWeb function has no ending, so there is a missing parenthesis "}" around line 105 (before the test cases start).

i fixed it now.

**Pull request merged**